### PR TITLE
doc: Add instructions on installing numpy

### PR DESCRIPTION
**Pull request by jalateras**

### CODE REVIEW

**Comment by jalateras** [comment]
I signed it

File: syntaxnet/README.md [hunk 1]
```diff
@@ -84,7 +84,9 @@ source. You'll need to install:
     *   upgrade to a supported version with `pip install -U protobuf==3.0.0b2`
 *   asciitree, to draw parse trees on the console for the demo:
     *   `pip install asciitree`
-
+*   numpy, pacage for scientific computing:
```
Comment (calberti) [inline syntaxnet/README.md]: *package

**Comment by jalateras** [comment]
Not sure on your workflow ... but do you folks want me to squash and force push to the branch?

**Comment by zhangjiulong** [comment]
Hi I follow the instruction and at the test step i got 11 passed and the last one missed.
The error log is like
'''
Executed 1 out of 12 tests: 11 tests pass and 1 fails locally.
There were tests whose specified size is too big. Use the --test_verbose_timeout_warnings command line option to see which ones these are.
'''
The I used the ' --test_verbose_timeout_warnings' parameter like this
'''
 bazel test syntaxnet/... util/utf8/... --test_verbose_timeout_warnings
```
and I got the log like this
```
//syntaxnet:reader_ops_test                                              FAILED in 2.2s
  /home/zhangjl/.cache/bazel/_bazel_zhangjl/83e8305c713f72c9599677dec7a62070/syntaxnet/bazel-out/local-opt/testlogs/syntaxnet/reader_ops_test/test.log

Executed 1 out of 12 tests: 11 tests pass and 1 fails locally.
```

I checked the log file '.cache/bazel/_bazel_zhangjl/83e8305c713f72c9599677dec7a62070/syntaxnet/bazel-out/local-opt/testlogs/syntaxnet/reader_ops_test/test.log'
bug I can't fine any error message.
The last one like this 
```
OK (skipped=1)
```
what's the matter？？
[test.txt](https://github.com/tensorflow/models/files/267205/test.txt)

**Pull request merged**